pkg/install/drbd/drbd_adapter.go

```diff
@@ -98,375 +104,537 @@ func HandelDRBDConfigs(clusterInstance *hwameistoriov1alpha1.Cluster) {
-										Path: "/etc/drbd.conf",
-										Type: &install.HostPathFileOrCreate,
-									},
+						},
+						{
+							Name: "etc-drbd-conf",
+							VolumeSource: corev1.VolumeSource{
+								HostPath: &corev1.HostPathVolumeSource{
+									Path: "/etc/drbd.conf",
+									Type: &install.HostPathFileOrCreate,
 								},
 							},
-							{
-								Name: "etc-drbd-d",
-								VolumeSource: corev1.VolumeSource{
-									HostPath: &corev1.HostPathVolumeSource{
-										Path: "/etc/drbd.d",
-										Type: &install.HostPathDirectoryOrCreate,
-									},
+						},
+						{
+							Name: "etc-drbd-d",
+							VolumeSource: corev1.VolumeSource{
+								HostPath: &corev1.HostPathVolumeSource{
+									Path: "/etc/drbd.d",
+									Type: &install.HostPathDirectoryOrCreate,
 								},
 							},
-							{
-								Name: "var-lib-drbd",
-								VolumeSource: corev1.VolumeSource{
-									HostPath: &corev1.HostPathVolumeSource{
-										Path: "/var/lib/drbd",
-										Type: &install.HostPathDirectoryOrCreate,
-									},
+						},
+						{
+							Name: "var-lib-drbd",
+							VolumeSource: corev1.VolumeSource{
+								HostPath: &corev1.HostPathVolumeSource{
+									Path: "/var/lib/drbd",
+									Type: &install.HostPathDirectoryOrCreate,
 								},
 							},
-							{
-								Name: "etc-modules-load",
-								VolumeSource: corev1.VolumeSource{
-									HostPath: &corev1.HostPathVolumeSource{
-										Path: "/etc/modules-load.d",
-										Type: &install.HostPathDirectoryOrCreate,
-									},
+						},
+						{
+							Name: "etc-modules-load",
+							VolumeSource: corev1.VolumeSource{
+								HostPath: &corev1.HostPathVolumeSource{
+									Path: "/etc/modules-load.d",
+									Type: &install.HostPathDirectoryOrCreate,
 								},
 							},
-							{
-								Name: "etc-sysconfig-modules",
-								VolumeSource: corev1.VolumeSource{
-									HostPath: &corev1.HostPathVolumeSource{
-										Path: "/etc/sysconfig/modules",
-										Type: &install.HostPathDirectoryOrCreate,
-									},
+						},
+						{
+							Name: "etc-sysconfig-modules",
+							VolumeSource: corev1.VolumeSource{
+								HostPath: &corev1.HostPathVolumeSource{
+									Path: "/etc/sysconfig/modules",
+									Type: &install.HostPathDirectoryOrCreate,
 								},
 							},
 						},
 					},
 				},
 			},
-		}
+		},
+	}
 
-		matched, err := regexp.Match("^rhel[78]$", []byte(distro))
-		if err != nil {
-			log.Errorf("Regexp match err: %v", err)
-			return adapterCreatedJobNum, err
-		}
-		if matched {
-			for i, container := range job.Spec.Template.Spec.Containers {
-				if container.Name == distro {
-					container.VolumeMounts = append(container.VolumeMounts, corev1.VolumeMount{
-						Name:      "centos-release",
-						MountPath: "/etc/centos-release",
-						ReadOnly:  true,
-					})
-					job.Spec.Template.Spec.Containers[i] = container
-				}
+	matched, err := regexp.Match("^rhel[78]$", []byte(distro))
```

Comment:
just a reminder, we have another issue about wrong idenfify system version - https://github.com/hwameistor/hwameistor-operator/issues/290

this can be fixed at another PR